### PR TITLE
Document how to modify imported plans

### PR DESCRIPTION
**Pull request by ZelenyMartin**
Finalizing test and documenting feature introduced in PR #1886

### CODE REVIEW

File: spec/plans/import.fmf [hunk 1]
```diff
@@ -27,6 +27,10 @@ description: |
     steps might be possible in the future but currently is not
     supported in any way.
 
+    Current only way to modify imported plan is via environment
+    variables. Variables defined in the plan override any variables
+    defined in the remote plan.
```
Comment (psss) [inline spec/plans/import.fmf:32]: Please, join the new paragraph with the preceding one and adjust `is not supported in any way` accordingly.

Comment (ZelenyMartin) [reply spec/plans/import.fmf:30]: Changed in amended commit.

Comment (psss) [reply spec/plans/import.fmf:30]: Thanks.

**Review by psss — APPROVED** [review]
Nice, thanks for finishing this. I've only added one word :)

**Pull request merged**